setup.cfg

```diff
@@ -0,0 +1,50 @@
+[metadata]
+name = requre
+url = https://github.com/packit-service/requre
+description = Library for testing python code what allows store output of various objects and use stored data for testing
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Red Hat
+author_email = packit@redhat.com
+license = MIT
+license_file = LICENSE
+classifiers =
+    Development Status :: 4 - Beta
+    Environment :: Console
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Topic :: Software Development
+    Topic :: Utilities
+keywords =
+    mocking
```

Comment:
This is something for @TomasTomecek !

Reply:
okay, I'll remove, but this is keyword and it is close to dynamic mocking :-)

Reply:
Personally, I have no problem with that...;-)